packages/agoric-cli/test/agops-oracle-smoketest.sh

```diff
@@ -64,14 +64,32 @@ agd query vstorage keys published.priceFeed
 agoric follow :published.priceFeed.ATOM-USD_price_feed.latestRound
 
 # Set it to $13 per ATOM
+# second round, second oracle has to be first this time (alternating start)
+PROPOSAL_OFFER=$(mktemp -t agops.XXX)
+bin/agops oracle pushPriceRound --price 14.0 --roundId 2 --oracleAdminAcceptOfferId "$ORACLE2_OFFER_ID" >|"$PROPOSAL_OFFER"
+agoric wallet send --offer "$PROPOSAL_OFFER" --from gov2 --keyring-backend="test"
 # second round, first oracle
 PROPOSAL_OFFER=$(mktemp -t agops.XXX)
-bin/agops oracle pushPriceRound --price 120000000 --roundId 2 --oracleAdminAcceptOfferId "$ORACLE_OFFER_ID" >|"$PROPOSAL_OFFER"
+bin/agops oracle pushPriceRound --price 12.0 --roundId 2 --oracleAdminAcceptOfferId "$ORACLE_OFFER_ID" >|"$PROPOSAL_OFFER"
 agoric wallet send --offer "$PROPOSAL_OFFER" --from gov1 --keyring-backend="test"
-# second round, second oracle
-PROPOSAL_OFFER=$(mktemp -t agops.XXX)
-bin/agops oracle pushPriceRound --price 14000000 --roundId 2 --oracleAdminAcceptOfferId "$ORACLE2_OFFER_ID" >|"$PROPOSAL_OFFER"
-agoric wallet send --offer "$PROPOSAL_OFFER" --from gov2 --keyring-backend="test"
 
 # see new price
 agoric follow :published.priceFeed.ATOM-USD_price_feed
+
+# Set it to $20 per ATOM
+PROPOSAL_OFFER=$(mktemp -t agops.XXX)
+bin/agops oracle pushPriceRound --price 15.0 --roundId 3 --oracleAdminAcceptOfferId "$ORACLE_OFFER_ID" >|"$PROPOSAL_OFFER"
+agoric wallet send --offer "$PROPOSAL_OFFER" --from gov1 --keyring-backend="test"
+# second oracle
+PROPOSAL_OFFER=$(mktemp -t agops.XXX)
+bin/agops oracle pushPriceRound --price 25.0 --roundId 3 --oracleAdminAcceptOfferId "$ORACLE2_OFFER_ID" >|"$PROPOSAL_OFFER"
+agoric wallet send --offer "$PROPOSAL_OFFER" --from gov2 --keyring-backend="test"
+
+# leave round unspecified for contract to suggest ($21.2)
```

Comment:
I'm struggling to find the mechanism that fills this in. hm. I saw a shape that made it look required.